clouddrift/adapters/gdp.py

```diff
@@ -0,0 +1,548 @@
+import numpy as np
+import pandas as pd
+from datetime import datetime
+import xarray as xr
+import urllib.request
+import concurrent.futures
+import re
+import tempfile
+from tqdm import tqdm
+import os
+import warnings
+
+GDP_VERSION = "2.00"
+GDP_DATA_URL = "https://www.aoml.noaa.gov/ftp/pub/phod/lumpkin/hourly/v2.00/netcdf/"
+GDP_FILENAME_PATTERN = "drifter_{id}.nc"
+GDP_TMP_PATH = os.path.join(tempfile.gettempdir(), "clouddrift", "gdp")
```

Comment:
Using a temporary directory means you have to download the files every time you want to recreate a subset ragged array. Maybe this could be an argument and have the default path a temporary one?

Reply:
Good idea, I agree

Reply:
Yes, I second this.

Reply:
Opened #94 to track this.